dev/create-release/release-build.sh

```diff
@@ -235,40 +240,46 @@ if [[ "$1" == "package" ]]; then
       -DzincPort=$ZINC_PORT 2>&1 >  ../binary-release-$NAME.log
     cd ..
 
-    if [[ -n $R_FLAG ]]; then
-      echo "Copying and signing R source package"
-      R_DIST_NAME=SparkR_$SPARK_VERSION.tar.gz
-      cp spark-$SPARK_VERSION-bin-$NAME/R/$R_DIST_NAME .
-
-      echo $GPG_PASSPHRASE | $GPG --passphrase-fd 0 --armour \
-        --output $R_DIST_NAME.asc \
-        --detach-sig $R_DIST_NAME
-      echo $GPG_PASSPHRASE | $GPG --passphrase-fd 0 --print-md \
-        SHA512 $R_DIST_NAME > \
-        $R_DIST_NAME.sha512
+    if [ -z "$AMPLAB_JENKINS" ]; then
+      if [[ -n $R_FLAG ]]; then
+        echo "Copying and signing R source package"
+        R_DIST_NAME=SparkR_$SPARK_VERSION.tar.gz
+        cp spark-$SPARK_VERSION-bin-$NAME/R/$R_DIST_NAME .
+
+        echo $GPG_PASSPHRASE | $GPG --passphrase-fd 0 --armour \
```

Comment:
I'd flip this though? I think we should need to copy but just not sign - otherwise the R and python package might not be saved from the build?